### ISSUE TITLE
Ghost Appliance needs MySQL 8 to update to latest version

### ISSUE THREAD

**Issue by jmadden91** [turnkeylinux/tracker#1795]
Hi guys, 

Thanks for your work with the TKL project. 

Just wanted to open an issue to flag Ghost V5 requires MySQL 8 rather than the MariaDB that the V17.1 appliance comes packaged with.
Updating is not possible with the MariaDB database.

Cheers

**Comment by JedMeister** [comment]
Yeah it's very unfortunate. We'll probably need to drop the Ghost appliance, unless we build (and support) MySQL ourselves.

See also [discussion about this on the forums](https://www.turnkeylinux.org/forum/general/wed-20221228-1430/ghost-db-configuration-blocks-upgrade)

**Comment by JedMeister** [comment]
FWIW I have done some more investigations and despite the scary warning on the Oracle MySQL website (where the apt config deb can be downloaded) the actual MySQL packages (specifically the 'community' ones) explicitly note in their licence files that they are open source (GPLv2).

As such I've updated the build code to install Oracle MySQL (instead of MariaDB): https://github.com/turnkeylinux-apps/ghost/pull/18

However, unfortunately whilst it successfully builds an ISO and Ghost appears to be running ok, it seems impossible to log in. It appears that things have changed a bit and our previous assumptions no longer apply. I assume that it is possible to manually tweak the DB to make it work, but after spending a ton of time on it and having little to no progress, I'm going to have to put it aside for now.

Perhaps we'll revisit this once we start work on v18.x? If anyone else wants to have a go at it, I'm more than happy to provide some coaching.

**Comment by JedMeister** [comment]
Kicking this can down the road...
